verification/curator-service/ui/src/components/AutomatedBackfill.tsx

```diff
@@ -0,0 +1,208 @@
+                            </Typography>
+                            <Typography
+                                className={classes.headerBlurb}
+                                data-testid="header-blurb"
+                                variant="body2"
+                            >
+                                Automatically ingest cases for a configured
+                                source over a specified date range.
+                            </Typography>
+                        </div>
+                        <Form>
+                            <Paper className={classes.allFormSections}>
+                                <div className={classes.formSection}>
+                                    <Source disableFreeSolo={true} />
+                                </div>
+                                <DateField
+                                    name="startDate"
+                                    label="First date to backfill"
+                                    required
+                                ></DateField>
+                                <DateField
+                                    name="endDate"
+                                    label="Last date to backfill"
+                                    required
+                                ></DateField>
+                            </Paper>
+                        </Form>
+                        {successMessage && (
+                            <MuiAlert
+                                className={classes.statusMessage}
+                                elevation={6}
+                                variant="filled"
+                                severity="success"
+                            >
+                                {successMessage}
+                            </MuiAlert>
+                        )}
+                        {errorMessage && (
+                            <MuiAlert
+                                className={classes.statusMessage}
+                                elevation={6}
+                                variant="filled"
+                                severity="error"
+                            >
+                                {errorMessage}
+                            </MuiAlert>
+                        )}
+                        <div className={classes.buttonBar}>
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                data-testid="submit"
+                                disabled={isSubmitting}
+                                onClick={submitForm}
+                            >
+                                Backfill source
+                            </Button>
+                            <Button
+                                className={classes.cancelButton}
+                                color="primary"
+                                disabled={isSubmitting}
```

Comment:
nit but can you disable the modal X button as well? do we even want to do that?

Reply:
We could find a way I'm sure, but an escape hatch is probably good to keep. This came up with the other modal designs talking with UX.